### ISSUE TITLE
Error after steam log in - type 'NoneType' has no len()

### ISSUE THREAD

**Issue by kaskii** [splewis/get5-web#36]
Hello,

I installed successfully and saw the front page. After Steam log in I get following message:
`Sorry, unexpected error: object of type 'NoneType' has no len()`

I found one issue with same error message but reinstalling db as suggested didn't help me. (dropped tables and ran `manager.py db upgrade` again). Installing requirements just says `Requirement already satisfied` on all lines.

[Log file](http://pastebin.com/raw/cLVsem1f)

- Linux version 4.4.0-45-generic (Ubuntu 5.4.0-6ubuntu1~16.04.2)
- Apache 2.4.18-2ubuntu3.1
- Python 2.7.11-1
- libapache2-mod-wsgi 4.3.0-1.1build1

**Comment by splewis** [comment]
Hm, that's a surprising error. I haven't managed to reproduce it yet. 

Can you describe your database setup? Are you using MySQL or another database?

**Comment by kaskii** [comment]
Thank you for your response. I'm using MariaDB 10.0.27. I actually resolved this issue just few minutes ago. I'm not 100% what went wrong in the first time but I guess there was some error on my end with the install, although I don't remember getting any error messages. 

Also after the "failed" install and getting that error, I just wiped everything (dropped database, deleted files, reverted apache2 vhost...). After reinstalling everything according the README, I still got the same message when opening the URL but this time I didn't even get to the front page to log in with steam. I noticed that in chromes incognito mode I did get to the front page as well as logged in successfully. 

It seems like some cached data or cookies were saved by my web browser which interfered even with the new installation of get5-web (I deleted whole get5-web folder from my system). After clearing them my second new installation was working.
